PowerEditor/src/TinyXml/tinyxml.cpp

```diff
@@ -1025,7 +1025,7 @@ void TiXmlUnknown::Print( FILE* cfile, int depth ) const
 {
 	for ( int i=0; i<depth; i++ )
 		generic_fprintf( cfile, TEXT("    ") );
-	generic_fprintf( cfile, TEXT("%s"), value.c_str() );
+	generic_fprintf( cfile, TEXT("%ls"), value.c_str() );
```

Comment:
I am wondering if `%ls` work for `fwprintf`.

My doubt comes from the following link:
https://stackoverflow.com/questions/26816547/whats-the-difference-between-printfs-printfls-wprintfs-and-wp

Reply:
Yes, it works in both `fprintf` and `fwprintf` to mean a `wchar_t*` argument for both GCC and MSC.

Reply:
@donho 
`The various printf and wprintf functions take a format string and optional arguments and produce a formatted sequence of characters for output. `
from https://docs.microsoft.com/en-us/cpp/c-runtime-library/format-specification-syntax-printf-and-wprintf-functions?view=msvc-160

and list of function using these format specification: https://docs.microsoft.com/en-us/cpp/c-runtime-library/stream-i-o?view=msvc-160

And this is why GCC has issue with `%s`, but MSC not
`An ls, lS, ws, or wS type specifier is synonymous with S in printf functions and with s in wprintf functions.`
from https://docs.microsoft.com/en-us/cpp/c-runtime-library/format-specification-syntax-printf-and-wprintf-functions?view=msvc-160#size-prefixes-for-printf-and-wprintf-format-type-specifiers